recipes/discasm/meta.yaml

```diff
@@ -1,38 +1,47 @@
+{% set name = "discasm" %}
 {% set version = "0.1.2" %}
+{% set build = "1" %}
 
 package:
-    name: discasm
-    version: {{ version }}
+  name: {{ name }}
+  version: {{ version }}
 
 source:
   fn: DISCASM-v{{ version }}.tar.gz
   url: https://github.com/DISCASM/DISCASM/releases/download/DISCASM-v{{ version }}/DISCASM-v{{ version }}.tar.gz
   md5: 571b2265018608298752f40f3f375b0f
 
 build:
-    number: 0
-    skip: true  # [osx or py3k]
-    # The reason for skipping osx is that trinity >=2.4 is not available in the osx channel
-    # The python code in DISCASM runs on python 2.7.
+  number: 1
+  skip: true  # [osx or py3k]
+  # The reason for skipping osx is that trinity >=2.4 is not available in the osx channel
+  # The python code in DISCASM runs on python 2.7.
 
 requirements:
-    build:
-    run:
-        - python
-        - perl
-        - star >=2.4
-        - trinity >=2.4
-        - oases >=0.2
+  build:
+  run:
+    - python
+    - pysam
+    - perl
+    - star >=2.4
+    - trinity >=2.4
+    - oases >=0.2
 
 test:
-    commands:
-        - DISCASM --help > /dev/null 2>&1
-        # DISCASM requires the environment variable TRINITY_HOME to be set.
-        # If TRINITY_HOME is not set before it starts, it uses "which" to find
-        # Trinity and sets TRINITY_HOME to be the directory where Trinity resides.
+  commands:
+    - DISCASM --help 2>&1
+    # DISCASM requires the environment variable TRINITY_HOME to be set.
+    # If TRINITY_HOME is not set before it starts, it uses "which" to find
+    # Trinity and sets TRINITY_HOME to be the directory where Trinity resides.
```

Comment:
That comment about `TRINITY_HOME` isn't really relevant for the recipe, but should rather be part of the software's documentation, e.g., readme file or wiki.

Reply:
ok. will remove.